scripts/publish-docker-images

```diff
@@ -123,9 +127,10 @@ push_multi-arch_manifest_docker_hub() {
 		# Replace the tags in multi-arch.yaml
 		sed -e "s/\${docker-image-tag}/${image_tag}/"  ../scripts/multi-arch.yaml > multi-arch-temp.yaml
 		# Generate the manifest list and push to docker hub
-		dryval ./manifest-tool push from-spec multi-arch-temp.yaml
+		MULTIARCH_IMAGE_SHA=$(dryval ./manifest-tool push from-spec multi-arch-temp.yaml)
```

Comment:
The return value is just a sha?
Based on this: https://github.com/estesp/manifest-tool/blob/6bcfccea0827b4ff930f2798440cfce6cd4f1318/push.go. Seems it has sha and length as output?

Reply:
You are right, fixed it.